CHANGES.md

```diff
@@ -1,5 +1,11 @@
 ## Change Log
 
+### Unreleased
+
+#### _Packaging_
+
+- Install `primer.json` with black. (#2153)
```

Comment:
Bummer ... so close!
```suggestion
- Install `primer.json` with black. (#2154)
```

Reply:
Might be worth it for mention this is only for black-primer (not general black users)?

Reply:
It should be the PR number, makes it easier to check automatically that it's correct.

Reply:
I updated via web.

Reply:
Our changelog policy to give the contributor the credit by linking the PR number. I was going to link our changelog policy but I just realized we don't mention that it's the PR number we're looking for...

I should fix that, sorry !

Reply:
That was non-obvious for me. ack